Makefile

```diff
@@ -0,0 +1,53 @@
+# system python interpreter. used only to create virtual environment
+PY = python3
+VENV = venv
+BIN=$(VENV)/bin
+
+# make it work on windows too
+ifeq ($(OS), Windows_NT)
+    BIN=$(VENV)/Scripts
+    PY=python
+endif
+
+.PHONY: venv clean install test artemis graphdb model
+
+## help: Show help
+help: Makefile
+	@printf "\n\033[1mUsage: make <TARGETS> ...\033[0m\n\n\033[1mTargets:\033[0m\n\n"
+	@sed -n 's/^## //p' $< | awk -F':' '{printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}' | sort | sed -e 's/^/  /'
+
+## venv: Create virtual environment and runs pip install
+$(VENV):
+	$(PY) -m venv $(VENV)
+	touch $(VENV)
```

Comment:
Why this touch?
Don't we need to activate the VENV with something like "source $(VENV)/bin/activate" ?

Reply:
Good catch.

Reply:
True, right now this does not make sense. The goal of this line is to make the makefile detect when it needs to run venv or not, but it will require further work. I'll update the PR.